src/constants/parseJson.ts

```diff
@@ -0,0 +1,45 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+// copied from vscode-azurefunctions extension
```

Comment:
Why is this up here? And why is it in the constants folder? Having a separate module for this seems fine, but I'd just put it in the utils folder instead.

Also please provide an actual link to where this is from

Reply:
Many of these files were re-used from sql-database-projects and as such, they were checked in like this. Will move and add link.